src/Orleans.Core.Abstractions/Orleans.Core.Abstractions.csproj

```diff
@@ -13,6 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="$(SystemRuntimeVersion)" />
```

Comment:
Similarly for other reviewers, we should eventually get rid of this dependency before we release 2.0 final.